cypress/integration/test_table.ts

```diff
@@ -37,10 +37,12 @@ describe("Tests Table", () => {
     });
   });
 
-  it("Should display No Data when given an invalid TaskID in the url", () => {
+  it("Should display error banner when given an invalid TaskID in the url", () => {
     cy.visit("/task/NO-SUCH-THANG/tests");
     cy.waitForGQL("GetTask");
-    cy.contains("No Data");
+    cy.dataCy("banner").contains(
+      "There was an error loading the task: GraphQL error:"
```

Comment:
nit. Bring this out to an external variable.

Reply:
also not necessary